_docs/tasks/security/authn-policy.md

```diff
@@ -168,6 +212,20 @@ spec:
 EOF
 ```
 
+and destination rule:
```

Comment:
What's this for? There's no namespace level DR before, why we need a per-service DR to disable it now?